pkg/sylog/sylog.go

```diff
@@ -20,7 +20,7 @@ import (
 	"strings"
 )
 
-const messageLevelEnv = "APPTAINER_MESSAGELEVEL"
+const MessageLevelEnv = "APPTAINER_MESSAGELEVEL"
```

Comment:
Instead of making this global, I think it would be better to have a function to get the message level.

In fact, for almost all environment variables we support either a SINGULARITY_ or APPTAINER_ prefix.  If it's not difficult to use the common function where that's done that would be better, even though I know it wasn't being done before.  For the test case where this variable is set I think it's fine to use the string `APPTAINER_MESSAGELEVEL` rather than a global variable name.

Reply:
just using pure string instead
https://github.com/apptainer/apptainer/pull/1569/commits/1a2003474ceaa5144f602b234aa4bd811f3d336b

Reply:
How about using nternal/pkg/util/env/env.go's GetenvLegacy to get the MESSAGELEVEL?

Reply:
reflected in this commit
https://github.com/apptainer/apptainer/pull/1569/commits/7f8566c352d1a5affa0cfdbef11bc61a0c9130b6
sylog/sylog_dummy could not use GetenvLegacy due to the cycle import.